src/components/containers/ContextContainer.tsx

```diff
@@ -1,8 +1,9 @@
-import { Container } from 'constate'
 import React from 'react'
+import { Container } from 'constate'
+import { LocaleData } from 'utils/locales'
 
-interface Context {
-  lang: string
+type Context = {
+  currentLocale: LocaleData
 }
```

Comment:
Maybe we should use `interface` still?

Reply:
I'm going to leave the `type` alias since we agreed that we might transition to using `type` over `interface` for component props and state, and that it's fine that both are used in the codebase for now. 😇